tests/lib/ocpp/v201/test_smart_charging_handler.cpp

```diff
@@ -206,29 +183,26 @@ class ChargepointTestFixtureV201 : public testing::Test {
 };
 
 TEST_F(ChargepointTestFixtureV201, K01FR03_IfTxProfileIsMissingTransactionId_ThenProfileIsInvalid) {
-    create_evse_with_id(DEFAULT_EVSE_ID);
     auto profile = create_tx_profile_with_missing_transaction_id(create_charge_schedule(ChargingRateUnitEnum::A));
-    auto sut = handler.validate_tx_profile(profile, *evses[DEFAULT_EVSE_ID]);
+    auto sut = handler.validate_tx_profile(profile, evse_manager->get_evse(DEFAULT_EVSE_ID));
 
     EXPECT_THAT(sut, testing::Eq(ProfileValidationResultEnum::TxProfileMissingTransactionId));
 }
 
-TEST_F(ChargepointTestFixtureV201, K01FR16_IfTxProfileHasEvseIdNotGreaterThanZero_ThenProfileIsInvalid) {
-    auto wrong_evse_id = STATION_WIDE_ID;
-    create_evse_with_id(wrong_evse_id);
-    auto profile = create_charging_profile(DEFAULT_PROFILE_ID, ChargingProfilePurposeEnum::TxProfile,
-                                           create_charge_schedule(ChargingRateUnitEnum::A), uuid());
-    auto sut = handler.validate_tx_profile(profile, *evses[wrong_evse_id]);
+// TODO: Review this test. TxProfileEvseIdNotGreaterThanZero does not make sense since you can't get an EvseInterface& with get_id()==0
```

Comment:
@christopher-davis-afs @couryrr-afs This is the PR I was talking about during the WG meeting. 

I had to disable this test since I could not find a logical way to fix it. Do you have any insights here?

Also comments in general are appreciated.